src/pages/trade/HorizontalPanel.tsx

```diff
@@ -95,12 +95,12 @@ export const HorizontalPanel = ({ isOpen = true, setIsOpen }: ElementProps) => {
       {
         value: InfoSection.Position,
         label: stringGetter({
-          key: showCurrentMarket ? STRING_KEYS.POSITION : STRING_KEYS.POSITIONS,
+          key: isTablet ? STRING_KEYS.POSITION : STRING_KEYS.POSITIONS,
```

Comment:
is the plan to remove this for mobile later on?

Reply:
hmm I don't think mobile has decided to cut single position view yet, cc @mike-dydx

Reply:
i meant mobile web!